### ISSUE TITLE
bemenu fails to run on wlroots 0.8.1

### ISSUE THREAD

**Issue by Wychmire** [Cloudef/bemenu#68]
bemenu works on wlroots 0.6 and doesn't do anything on wlroots 0.8.1. wlroots 0.7 and 0.8 are untested.

When run from the console the binary launches but does nothing. e.g. `echo "foo" | bemenu` on x11 will show something like
```
bemenu             foo
```
at the top of your screen. However on wlroots 0.8.1 nothing happens.

Since there isn't any output I'm not sure how helpful this bug report is, but at least you know about it now. Can I compile with debug messages enabled or something?

**Comment by Wychmire** [comment]
Things get weirder: I had downgraded my wlroots and sway versions to 0.6 and 1.1 respectively because I wanted to be able to use bemenu to launch applications. To test Wayfire 0.3 I re-updated wlroots to 0.8.1 (and Sway to 1.2) and then launched Sway. Bemenu worked there. Then I exited and launched Wayfire, bemenu did _not_ work.

I've tried logging out of my tty session/logging back in and rebooting, bemenu still doesn't work on Wayfire, still works on Sway.
